.devcontainer/devcontainer.json

```diff
@@ -1,39 +1,37 @@
+// For more details, see https://aka.ms/devcontainer.json.
 {
-  "name": "Mastodon",
-  "dockerComposeFile": "docker-compose.yml",
-  "service": "app",
-  "workspaceFolder": "/mastodon",
+	"name": "Mastodon",
```

Comment:
Running `yarn format` should address the whitespace changes

Reply:
Thanks for the pointer, updated!